src/openassetio-core/include/openassetio/Context.hpp

```diff
@@ -16,77 +16,32 @@ inline namespace OPENASSETIO_CORE_ABI_VERSION {
 OPENASSETIO_DECLARE_PTR(Context)
 
 /**
- *  The Context object is used to convey information about the calling
- *  environment to a @ref manager. It encapsulates several key access
- *  properties, as well as providing additional information about the
- *  @ref host that may be useful to the @ref manager.
+ * The Context object is used to convey information about the calling
+ * environment to a @ref manager.
  *
- *  A Manager will also use this information to ensure it presents the
- *  correct UI, or behavior.
+ * It holds additional information about the @ref host that may be
+ * useful to the @ref manager. Optionally, it holds arbitrary, opaque
+ * (to the host), serializable manager state, which can be used by the
+ * manager plugin to tie together related calls whilst remaining
+ * stateless itself.
```

Comment:
[Consider] "whilst remaining stateless itself." sees like redundant information as far as the implementor is concerned.

Reply:
Removed in a01dec12